From 7ca3e5279d05fceb1e8a043c6f5b6f58dea3ed38 Mon Sep 17 00:00:00 2001
From: Marko Kreen <markokr@gmail.com>
Date: Thu, 3 Sep 2015 15:21:24 +0300
Subject: [PATCH] Remove too early set of auth_user

When query returns 0 rows (user not found),
this user stays as login user...

Should fix #69.
---
 src/client.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/client.c b/src/client.c
index 99615a0e3..2b01ff2cc 100644
--- a/src/client.c
+++ b/src/client.c
@@ -83,7 +83,6 @@ static void start_auth_request(PgSocket *client, const char *username)
 	int res;
 	PktBuf *buf;
 
-	client->auth_user = client->db->auth_user;
 	/* have to fetch user info from db */
 	client->pool = get_pool(client->db, client->db->auth_user);
 	if (!find_server(client)) {
